src/Repository/index.js

```diff
@@ -61,17 +71,17 @@ class Repository {
         return multipleFilesByPath
     }
 
-    async getHeadRef(defaultBranch) {
+    async getHeadRef(branchName) {
         const result = await this.github.git.getRef({
             owner: this.owner,
             repo: this.repo,
-            ref: `heads/${defaultBranch}`,
+            ref: `heads/${branchName}`,
         })
         return result.data.object.sha
     }
 
-    async createBranch({ branchName, defaultBranch }) {
-        const fromSha = await this.getHeadRef(defaultBranch)
+    async createBranch(branchName) {
+        const fromSha = await this.getHeadRef(this.defaultBranch)
```

Comment:
[nb] can we leave this as a parameter? `createBranch({ newBranchName, fromBranchName })`